crates/bevy_text/src/lib.rs

```diff
@@ -32,9 +40,19 @@ pub struct TextPlugin;
 
 impl Plugin for TextPlugin {
     fn build(&self, app: &mut AppBuilder) {
-        app.add_asset::<Font>()
-            .add_asset::<FontAtlasSet>()
-            .init_asset_loader::<FontLoader>()
-            .add_resource(DefaultTextPipeline::default());
+        app.add_stage_before(
+            bevy_app::stage::POST_UPDATE,
+            stage::TEXT2D,
+            SystemStage::parallel(),
+        )
+        .add_asset::<Font>()
+        .add_asset::<FontAtlasSet>()
+        .init_asset_loader::<FontLoader>()
+        .add_resource(DefaultTextPipeline::default())
+        .add_system_to_stage(stage::TEXT2D, text2d_system.system())
```

Comment:
I think we can remove this custom stage and add it to POST_UPDATE. That works for me, and it should increase parallelism potential.

Reply:
Done!  Would you like me to also update `bevy_ui` use POST_UPDATE instead of a custom stage?

Reply:
hmm for now lets not, just because right now there are a number of systems in the UI stage and i dont want to risk breaking things. im down to give it a try in a separate pr though.